src/main/resources/META-INF/plugin.xml

```diff
@@ -99,5 +99,10 @@
                 <!--                <override-text place="GitMacheteToolbar"/>-->
             </action>
         </group>
+
+        <!-- TODO (#274): consider excluding this action from the production builds of the plugin -->
+        <action id="GitMachete.OpenTabAction"
```

Comment:
This action might actually be useful. Idea log as an example:

![image](https://user-images.githubusercontent.com/19799111/82117021-e8bab100-976d-11ea-9989-4876ab8eed53.png)
 
Commented in https://github.com/VirtusLab/git-machete-intellij-plugin/issues/274#issuecomment-629621134 too.